engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -50,11 +55,22 @@
     private static final Logger logger = LoggerFactory.getLogger(FBO.class);
 
     public int fboId;
-    public int colorBufferTextureId;
-    public int depthStencilTextureId;
+    public enum BaseFboBuffer {
```

Comment:
If other usages may arise (in different classes), implementing an interface such as `FboBufferType` might be of advantage here. This would allow contributors to provide or lookup types that are not listed in `BaseFboBuffer`.